pkg/config/user_config.go

```diff
@@ -403,6 +403,7 @@ type KeybindingBranchesConfig struct {
 	SetUpstream            string `yaml:"setUpstream"`
 	FetchRemote            string `yaml:"fetchRemote"`
 	SortOrder              string `yaml:"sortOrder"`
+	Archive                string `yaml:"archive"`
```

Comment:
We could consider putting it in `Universal` instead; it's "almost" universal (and we might consider offering it in the subcommits view too). I find it very unlikely that users will want to map it to different keys for branches and commits.